src/Microsoft.SqlTools.ServiceLayer/LanguageServices/LanguageService.cs

```diff
@@ -230,6 +230,9 @@ public void InitializeService(ServiceHost serviceHost, SqlToolsContext context)
             // Register the file open update handler
             WorkspaceService<SqlToolsSettings>.Instance.RegisterTextDocOpenCallback(HandleDidOpenTextDocumentNotification);
 
+            // Register the IntelliSense rebuild handler
+            WorkspaceService<SqlToolsSettings>.Instance.RegisterRebuildIntelliSenseCallback(HandleRebuildIntelliSenseNotification);
```

Comment:
We don't need this indirection - we should just call `serviceHost.SetRequestHandler(RebuildIntelliSenseNotification.Type,, HandleRebuildIntelliSenseNotification)` instead. We already handle most other requests here (except global events that are common to the workspace and other stuff)